config/helpers.php

```diff
@@ -549,10 +472,6 @@ function site()
 if (Helpers::hasOverride('size') === false) { // @codeCoverageIgnore
 	/**
 	 * Determines the size/length of numbers, strings, arrays and countable objects
-	 *
-	 * @param mixed $value
-	 * @return int
-	 * @throws \Kirby\Exception\InvalidArgumentException
```

Comment:
Why is this line removed?

Reply:
We discussed that we only use the `@throws` tag for exceptions thrown in scope.

Reply:
That makes sense. At least it's consistent then. For users it's a lot less useful, but without tools it will be incredibly hard to track all exceptions to every public method where they could be thrown.